lib/user.rb

```diff
@@ -0,0 +1,27 @@
+class User
+
+  attr_reader :name, :jokes
+
+  def initialize(name)
+    @name = name
+    @jokes = []
+  end
+
+  def learn(joke)
+    @jokes << joke
+  end
+
+  def tell(jokester, joke)
+    jokester.learn(joke)
```

Comment:
really excellent job, using the jokester (😄 ), which is also an instance of a user, and calling the `learn` method you already built for a user.  perfect!